includes/api/class-wc-admin-rest-reports-products-stats-controller.php

```diff
@@ -213,6 +220,28 @@ public function get_item_schema() {
 		return $this->add_additional_fields_schema( $schema );
 	}
 
+	/**	
+	 * Set the default results to 0 if API returns an empty array
+	 * 
+	 * @param $results mixed
+	 * @param $args array
+	 * @return object
+	 */
+	public function set_default_report_data( $results, $args ) {
```

Comment:
`$args` is not needed here, right?

```suggestion
	public function set_default_report_data( $results ) {
```

Reply:
You're right, we can remove this.